src/main/webapp/app/core/navbar/global-search/components/modal/searchable-entity-item/searchable-entity-item.component.html

```
@@ -0,0 +1,37 @@
+<div
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)